Vague error code 65 from xcodebuild

I used the install script for Mac OS X to configure NativeScript and Android works just fine.  However, trying to run the bare bones app (`tns create`) crashes with iOS.  The emulator runs but the build fails with a vague error code of 65.  Here is the output of `tns run ios`:

```
Traceback (most recent call last):
  File "./metadata-generation-build-step", line 100, in <module>
    generate_metadata(arch)
  File "./metadata-generation-build-step", line 89, in generate_metadata
    error_file.write(error_stream_content)
TypeError: write() argument must be str, not bytes
Command /bin/sh failed with exit code 1

** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution NativeScript\ PreBuild build/demo.build/Debug-iphonesimulator/demo.build/Script-C97FD7AC1ADE5369004DB2A4.sh
(1 failure)
Command xcodebuild failed with exit code 65
```

Where do I start looking for the problem in my configuration or is this a bug?  I'm using OS X 10.11.4, NativeScript 2.0.0, and Xcode 7.3.

--- a/build/scripts/metadata-generation-build-step
+++ b/build/scripts/metadata-generation-build-step
@@ -79,7 +79,7 @@ def generate_metadata(arch):
     generator_call.extend(other_cflags_parsed)  # OTHER_CFLAGS
     generator_call.extend(preprocessor_defs_parsed)  # GCC_PREPROCESSOR_DEFINITIONS
 
-    child_process = subprocess.Popen(generator_call, stderr=subprocess.PIPE)
+    child_process = subprocess.Popen(generator_call, stderr=subprocess.PIPE, universal_newlines=True)
     sys.stdout.flush()
     error_stream_content = child_process.communicate()[1]
 

